tracing/src/main/java/org/axonframework/extensions/tracing/TracingQueryGateway.java

```diff
@@ -99,7 +100,13 @@ public static Builder builder() {
                                           ResponseType<R> responseType,
                                           long timeout,
                                           TimeUnit timeUnit) {
-        return delegate.scatterGather(queryName, query, responseType, timeout, timeUnit);
+        QueryMessage qry = new GenericQueryMessage(query, queryName, responseType);
```

Comment:
The same "is this a message?" check should be done here.

Reply:
Updated as well.